src/Sylius/Bundle/ReviewBundle/Resources/config/doctrine/model/Review.orm.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:gedmo="http://gediminasm.org/schemas/orm/doctrine-extensions-mapping">
+
+    <mapped-superclass name="Sylius\Component\Review\Model\Review" table="sylius_review">
+        <id name="id" type="integer" column="id">
+            <generator strategy="AUTO"/>
+        </id>
+        <field name="title" type="string" column="title"/>
+        <field name="rating" type="integer" column="rating"/>
+        <field name="comment" type="text" column="comment"/>
```

Comment:
Anyway, it would be awesome to have feature, that checks whether review is verified (eg. was sent by customer, that had bought that product earlier).